modules/core/shared/src/main/scala/lucuma/core/math/WavelengthDither.scala

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math
+
+import cats.Order
+import coulomb.Quantity
+import lucuma.core.math.units.Angstrom
+import lucuma.core.math.units.Micrometer
+import lucuma.core.math.units.Nanometer
+import lucuma.core.math.units.Picometer
+import lucuma.core.optics.Format
+import monocle.Iso
+
+import java.math.RoundingMode
+
+
+/**
+ * A wavelength dither is a small offset (positive or negative) intended to be
+ * applied to a wavelength, such as the observing wavelength.  It is therefore a
+ * small distance and not a `Wavelength` per se.  A `Wavelength + WavelengthDither`
+ * should produce a new `Wavelength`.
+ */
+opaque type WavelengthDither = Quantity[Int, Picometer]
+
+object WavelengthDither {
```

Comment:
Would you ever convert `Waveleng` to `WavelengthDither`?

Reply:
It doesn't occur to me when you ever would have a `Wavelength` value that serves as an offset.